spring-integration-ftp/src/main/java/org/springframework/integration/ftp/filters/FtpLastModifiedFileListFilter.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2015-2023 the original author or authors.
```

Comment:
This class was really created in the `2023`. Therefore copyright is just for the current year.